### PR TITLE
Only includes css necessary for this part of test

### PR DESCRIPTION
**Pull request by arrowarchive**
I tried working with grids, but I feel like I did something wrong here. Also, I have to remove the hardcoded width, forgot to do that. Also, I may have accidentally made a branch called "checkout". Please delete that branch if it exists.

### CODE REVIEW

**Review by ebeshero — COMMENTED** [review]
@arrowarchive You've set up grid here, but I don't see corresponding changes in the HTML file. I opened your rachelScaling branch files locally and I see that some width properties are still encoded in attributes on your HTML `` elements. For CSS grid (or flexbox) to function properly, we don't want conflicting systems for controlling the layout and scaling. We should **remove** all the hard-coded `@width` attributes in the HTML file.

**Comment by ebeshero** [comment]
@arrowarchive Let me test a minute here to make sure your CSS properties are also properly speaking to corresponding HTML elements. I might push a new commit to this branch for us to review.

File: docs/web/experimental.css
Comment (ebeshero) [inline docs/web/experimental.css:117]: @arrowarchive Typically an `` file is nested deep inside container elements. It is the *container* elements that take the `display:grid` CSS property. You need to set up a grid on a container element higher up the tree.

Comment (ebeshero) [reply docs/web/experimental.css:117]: @arrowarchive This is to say, I believer your `display:grid` properties are sitting on the wrong elements in this CSS file.

**Review by ebeshero — CHANGES_REQUESTED** [review]
@arrowarchive I'm seeing several coding issues here. Let me try to push some changes back to you to give you a better idea of how grid should work. I may not get this just the way you want it, but my goal is to help you better understand what you're doing with this CSS and its corresponding HTML.

**Comment by ebeshero** [comment]
@am0eba-byte Let's not merge this in to main just yet til we've done some rounds of revision in this branch...I'm about to push some commits here for @arrowarchive to pull in and review.

**Review by ebeshero — APPROVED** [review]

**Comment by ebeshero** [comment]
@arrowarchive @am0eba-byte I pushed some modifications to the HTML and CSS
*  I set up some clear container elements in the HTML (wrapping both `` elements in `<figure>` elements, and renaming the container container around the sidebar and main elements to be `<div class="grid">`.)  
 * I set up CSS grid to on the `<div class="grid">` element and on the `<div class="sidebar">` elements to control the proportion and scaling of their contents. 
 * I left lots of comments in the CSS to show how to make the images scale as you widen or shrink the browser window. 
 
 Hope this helps! Pull this in if you're satisfied with it, or keep tinkering. :-)

File: docs/web/experimental.css
Comment (ebeshero) [inline docs/web/experimental.css:75]: @arrowarchive You definitely don't need sticky positioning with CSS grid set up properly on a container element. This was removed.

File: docs/web/experimental.css
Comment (ebeshero) [inline docs/web/experimental.css:77]: @arrowarchive I changed your div.flex to div.grid, since this code is meant to implement CSS grid.

**Review by ebeshero — COMMENTED** [review]
@arrowarchive Hope these updates + comments in your CSS code help you to understand CSS grid a little better!

File: docs/web/experimental.css
Comment (ebeshero) [inline docs/web/experimental.css:99]: Set `<figure>` elements to be containers of your `` elements that you wish to scale. Give the container `<figure>` a relative width (here expressed in percentage).

File: docs/web/experimental.css
Comment (ebeshero) [inline docs/web/experimental.css:107]: @arrowarchive The two properties above (`object-fit:contain;` and `width:100%`) work together to make the image scale within the `<figure>` container element.

Comment (arrowarchive) [reply docs/web/experimental.css:107]: @ebeshero sorry for taking so long to respond! I see what you are trying to go for. I originally had the profile picture scaled to half its size for my own view, but if this code allows it to be scaled depending on the screen size and the contents of the screen, that makes more sense. 

As for what changed? I should have specified that it was the CSS that changed, not the HTML. I should have made that clearer, and I apologize for that.

File: docs/web/experimental.css
Comment (am0eba-byte) [reply docs/web/experimental.css:75]: Well it depends on how she wants the sidebar to look and behave.. she might want the sidebar to scroll down the side as the user scrolls so that it is always visible. In that case, I think she needs to keep the sticky on the sidebar. 

@arrowarchive you do still need to set the number of grid rows and columns you want your container elements to span across your page. See my link to the display:grid tutorial in my other comment!

File: docs/web/experimental.css
Comment (am0eba-byte) [inline docs/web/experimental.css]: @arrowarchive take a look at this: [A Very Handy, Illustrative, and Complete Guide to CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/)

**Review by am0eba-byte — CHANGES_REQUESTED** [review]
you might want to read up on [CSS grids](https://css-tricks.com/snippets/css/complete-guide-grid/) a bit more before merging this. You're getting the hang of it though!

**Review by am0eba-byte — APPROVED** [review]
good stuff!

**Review by am0eba-byte — APPROVED** [review]

**Comment by ebeshero** [comment]
@arrowarchive @am0eba-byte I'm merging this branch into main now, so we can see it publish to GitHub Pages from this repo! :-) This is fun... I see we may want to reconsider whether `position:sticky` should persist! :-) 

@arrowarchive Let's keep your branch alive and have you **rebase** it since you'll probably want to do more tinkering here.

**Comment by ebeshero** [comment]
@arrowarchive Also, yes, we'll talk about rebasing and what it does when we meet.

**Pull request merged**
